site/content/docs/howto/supervisor/configure-supervisor-federationdomain-idps.md

```diff
@@ -0,0 +1,403 @@
+---
+title: Configure Identity Providers (IDPs) on a FederationDomain
+description: Learn how to use one or more identity providers, and identity transformations and policies, on a FederationDomain.
+cascade:
+  layout: docs
+menu:
+  docs:
+    name: IDPs on FederationDomains
+    weight: 20
+    parent: howto-configure-supervisor
+---
+
+This guide explains how to associate one or more external identity providers (IDPs) with a FederationDomain.
+It also details how to configure identity transformations and identity policies for those identity
+providers.
+
+## Prerequisites
+
+This how-to guide assumes that you have already [installed the Pinniped Supervisor]({{< ref "install-supervisor" >}})
+and have already read the guide about how to [configure the Supervisor as an OIDC issuer]({{< ref "configure-supervisor" >}}).
+
+This guide focuses on the use of the `spec.identityProviders` setting on the FederationDomain CRD.
+
+## Summary
+
+External identity providers may be configured in the Supervisor by creating OIDCIdentityProvider,
+ActiveDirectoryIdentityProvider, or LDAPIdentityProvider resources in the same namespace as the Supervisor.
+
+There are two ways to configure which of these external identity providers shall be used by a FederationDomain.
+
+1. When there is no `spec.identityProviders` configured on a FederationDomain, then the FederationDomain will use
```

Comment:
nit: I tend to want to lead this sentence with `If there is exactly one IdentityProvider configured in the supervisor namespace, and there is no spec.identityProviders...` just to put the key point of backwards identity first.

(nit: meaning, not a big deal, feel free to ignore/resolve).

Reply:
It is first to be symmetrical with the leading sentence of the second bullet point. The difference between the two cases are whether or not you explicitly specify `spec.identityProviders`.